### PR TITLE
Change Pages to Static

### PR DESCRIPTION
**Pull request by VishalAiely**
Changes
---
- Changed `/chapter/index` to statically generate with revalidate
- Changed `/journal` and `/journal/[id]` to statically generate
- Changed the production URL so that frontend code can use it
- Made chapter-specific resources responsive

Minor
---
- Fixed empty body blog post being posted and removed extra category options

### CODE REVIEW

**Review by rluberto — APPROVED** [review]
All looks good to me!

File: pages/chapters/[name].tsx [hunk 1]
```diff
@@ -261,6 +267,7 @@ export async function getStaticProps(context: GetStaticPropsContext) {
             officers: JSON.parse(JSON.stringify(officers)) as Officer[],
             resources: JSON.parse(JSON.stringify(resources)) as Resource[],
         },
+        revalidate: 15,
```
Comment (kemalfidan) [inline pages/chapters/[name].tsx]: Can we extract the revalidate params to the urls or config file in utils? I put them in the urls file for now but we can have a separate place for globals from the urls.

File: pages/journal/[id].tsx [hunk 2]
```diff
@@ -1,159 +1,172 @@
...
+}
+
+export async function getStaticPaths() {
+    const posts: JournalEntry[] = await getJournalEntriesByReviewStatus(true);
+
+    const paths = posts.map(post => ({
+        params: { id: post.id },
+    }));
+
+    return { paths, fallback: false };
```
Comment (kemalfidan) [inline pages/journal/[id].tsx]: Do we want fallback to be true here since new journals might be added after the build?

**Review by kemalfidan — APPROVED** [review]
Looking good, thanks for making the data fetching changes. Didn't know getStaticProps() could take in a context but looks like it can! Also left some minor comments to be addressed

**Pull request merged**